### PR TITLE
BDRSPS-779 Added basic files for the site visit template.

### PR DESCRIPTION
**Pull request by joecrowleygaia**
Also created a template generation method which can be invoked to create blank csv template files from schemas in future.

### CODE REVIEW

File: abis_mapping/templates/survey_site_visit_v2/README.md
Comment (Lincoln-GR) [inline abis_mapping/templates/survey_site_visit_v2/README.md:1]: Should this new directory be `/survey_site_visit_data_v2/` to be consistent with the others?

File: abis_mapping/templates/survey_site_visit_v2/metadata.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	"label": "Systematic Survey Site Visit Data Template",
+	"version": "2.0.0",
+	"description": "A template for systematic survey site visit data",
+	"biodiversity_type": "Systematic Survey Site Visit Data",
+	"spatial_type": "Point, line, polygon",
+	"file_type": "CSV",
+	"sampling_type": "systematic survey",
+	"template_url": "https://raw.githubusercontent.com/gaiaresources/abis-mapping/main/abis_mapping/templates/survey_site_visit_data/survey_site_visit_data.csv",
+	"schema_url": "https://raw.githubusercontent.com/gaiaresources/abis-mapping/main/abis_mapping/templates/survey_site_visit_data/schema.json",
+	"instructions_url": "https://raw.githubusercontent.com/gaiaresources/abis-mapping/main/abis_mapping/templates/survey_site_visit_data/instructions.pdf"
```
Comment (Lincoln-GR) [inline abis_mapping/templates/survey_site_visit_v2/metadata.json:12]: should these urls match the name of this new directory?

**Review by Lincoln-GR — COMMENTED** [review]
looks good, 2 comments about the new directory

File: abis_mapping/templates/survey_site_visit_v2/metadata.json (hunk 1 above)
Comment (joecrowleygaia) [reply abis_mapping/templates/survey_site_visit_v2/metadata.json:12]: yep, good pickup

File: abis_mapping/templates/survey_site_visit_v2/README.md
Comment (joecrowleygaia) [reply abis_mapping/templates/survey_site_visit_v2/README.md:1]: fixed

**Review by Lincoln-GR — APPROVED** [review]

**Pull request merged**